src/library/Uno.Themes.WinUI.Markup/Theme/CheckBox.cs

```diff
@@ -0,0 +1,227 @@
+﻿using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+using Microsoft.UI.Xaml;
+using MUXCornerRadius = Microsoft.UI.Xaml.CornerRadius;
+using Microsoft.UI.Text;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class CheckBox
+		{
+			[ResourceKeyDefinition(typeof(double), "CheckBoxFocusAreaSize")]
+			public static ResourceValue<double> FocusAreaSize => new("CheckBoxFocusAreaSize", true);
+
+			public static class Resources
+			{
+				public static class Background
+				{
+					[ResourceKeyDefinition(typeof(Brush), "CheckBoxBackgroundUnchecked")]
+					public static ResourceValue<Brush> Unchecked => new("CheckBoxBackgroundUnchecked", true);
```

Comment:
Looks like `ResourceValue` was removed in latest master or these should be changed to `ThemeResourceKey`

Reply:
I updated the file, but I wasn't sure about `Theme` vs `Static` for some of the keys. Please take a look @kazo0, thanks!

Reply:
@silviuo Basically where we passed `true` to the constructor of `ResourceValue`, we should now use `ThemeResourceKey`, otherwise we use `StaticResourceKey`

Reply:
Looks good to me!